graphql/util.go

```diff
@@ -59,40 +60,48 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, sc data.Connector, taskID string, isActive bool) (*restModel.APITask, error) {
+func setScheduled(ctx context.Context, sc data.Connector, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
 	usr := mustHaveUser(ctx)
-	t, err := task.FindOneId(taskID)
+	tasks, err := task.FindAll(db.Query(task.ByIds(taskIDs)))
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-	if t == nil {
-		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task %s not found", taskID).Error())
+	if tasks == nil {
+		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task IDs %s not found", strings.Join(taskIDs, ",")).Error())
```

Comment:
An array of taskIds would not make for a useful/readable error message Since now there could be thousands of taskIds. Is there something else we could return?

Reply:
Might be good in this case to determine which taskIds weren't found and just return those (or at least log them, since yeah there could still be too many to return to the user)